SimulationRuntime/c/simulation/solver/stateset.c

```diff
@@ -345,64 +345,60 @@ void printStateSelectionInfo(DATA *data, STATE_SET_DATA *set)
   messageClose(LOG_DSS);
 }
 
-/*! \fn stateSelection
+/*! \fn stateSelectionSet
  *
- *  function to select the actual states
+ *  function to select the actual states for an individual stateSet
  *
  *  \param [ref] [data]
  *  \param [in]  [reportError]
  *  \param [in]  [switchStates] flag for switch states, function does switch only if this switchStates = 1
- *  \return ???
+ *  \return      [globalres] flag for discontinuous timestep in the case of a state switch
```

Comment:
`setIndex` is missing and `globalres` is input and not return value of the function